Cargo.toml

```diff
@@ -31,7 +31,7 @@ snafu = "0.8"
 strum = { version = "0.27", features = ["derive"] }
 tokio = { version = "1.40", features = ["full"] }
 tracing = "0.1"
-url = { version = "2.5.2" }
+url = { version = "2.5.7" }
```

Comment:
LGTM, but just FYI that this change was not needed. This is implicitly a range, so ">= 2.5.7 and < 2.6.0". The version in Cargo.toml is the one that is actually used for compilation.
For this reason we normally only put "major.minor" in binary Cargo.tomls and let Cargo.lock take of the patch level (see all the other dependencies above and below)